boards/arm/stm32l562e_dk/Kconfig.defconfig

```diff
@@ -45,4 +46,74 @@ config USE_DT_CODE_PARTITION
 
 endif # TRUSTED_EXECUTION_NONSECURE
 
+
+if LVGL
+
+config DISPLAY
+	default y
+
+config LVGL_DISPLAY_DEV_NAME
+	default "DISPLAY"
+
+config LVGL_HOR_RES_MAX
+	default 240
+
+config LVGL_VER_RES_MAX
+	default 240
+
+config LVGL_VDB_SIZE
+	default 16
+
+config LVGL_BITS_PER_PIXEL
+	default 16
+
+choice LVGL_COLOR_DEPTH
+	default LVGL_COLOR_DEPTH_16
+endchoice
+
+config KSCAN
+	default y
+
+config LVGL_POINTER_KSCAN
+	default y
+
+config LVGL_POINTER_KSCAN_DEV_NAME
+	default "FT5336"
+
+# Original orientation of the display needs to invert Y only
+
+config LVGL_POINTER_KSCAN_SWAP_XY
+	default n
```

Comment:
Why this ?

This is already a `default n` here:
https://github.com/zephyrproject-rtos/zephyr/blob/9c74027a7bbdc7348441ef2144a8c198c4e13d45/lib/gui/lvgl/Kconfig.input#L27-L29

Reply:
I must admit that this is probably some remains of when I ported the tft shield to the ST board, I had to disable this flag to properly align the axis of the touch controller. 

Is that a bad idea to leave it in the board def to show the user/board maintainers that swapping should be disabled ?

Reply:
I removed these config values since it is already default in LVGL